t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -380,17 +381,20 @@ sub wait_for_element {
     return $element;
 }
 
+sub kill_webapi() {
```

Comment:
Please rename to `stop_webapi` as both "kill" is a not necessarily nice word and second it's actually not "killing" the process following the POSIX definition, just "terminating" :)

EDIT: But *actually* this is just a duplication of `stop_service $webapi`, see t/lib/OpenQA/Test/Utils.pm::200